apps/ocsp.c

```diff
@@ -1180,8 +1180,10 @@ static char **lookup_serial(CA_DB *db, ASN1_INTEGER *ser)
     bn = ASN1_INTEGER_to_BN(ser, NULL);
     OPENSSL_assert(bn);         /* FIXME: should report an error at this
                                  * point and abort */
-    if (BN_is_zero(bn))
+    if (BN_is_zero(bn)) {
         itmp = OPENSSL_strdup("00");
+        OPENSSL_assert(itmp);
+    }
     else
```

Comment:
Out coding style mandates {} around the else part and the else on the same line as the if's }.

Reply:
OK, I have corrected it.